ci-scripts/docker/Dockerfile.mme.ci.ubuntu18

```diff
@@ -97,7 +97,7 @@ RUN ldconfig
 
 # Copy pre-built binaries for MME and SCTPD
 WORKDIR /magma-mme/bin
-COPY --from=magma-mme-builder $C_BUILD/oai/oai_mme/mme oai_mme
+COPY --from=magma-mme-builder $C_BUILD/core/oai/oai_mme/mme oai_mme
 COPY --from=magma-mme-builder $C_BUILD/sctpd/sctpd .
```

Comment:
@amarpad 

Here --> you still have the old location for `sctpd` if I am correct?